### DISCUSSION TITLE
Node Problem Detector cannot patch resource "nodes/status" in API group ""

### DISCUSSION THREAD

**Discussion by cmwatts1974** [open-policy-agent/community#257] [Community]
Running into issue where it will not do a patch.  Anyone have any ideas. Clusterrole and Clusterrolebinding in place.

**Comment by srenatus** [comment]
Please provide more information. Is this using GK or plain OPA? What have you tried? Can you share some CRDs or something?

While I'm probably not able to help myself, I'm fairly certain that being this brief, it's hard for anyone to help you there 😅

**Comment by cmwatts1974** [comment]
I have it install on Kubernetes in eks AWS.  I would appreciate any insight.  This is the error I am receiving...

 1 manager.go:162] failed to update node conditions: nodes "x.x.x.x" is forbidden: User "system:serviceaccount:kube-system:node-problem-detector" cannot patch resource "nodes/status" in API group "" at the error: stream error: stream ID 213; INTERNAL_ERROR; received from peer
